lang/src/test/kotlin/org/partiql/lang/eval/evaluatortestframework/ExprValueStrictEquals.kt

```diff
@@ -0,0 +1,146 @@
+package org.partiql.lang.eval.evaluatortestframework
+
+import org.partiql.lang.eval.ExprValue
+import org.partiql.lang.eval.ExprValueType
+import org.partiql.lang.eval.bigDecimalValue
+import org.partiql.lang.eval.booleanValue
+import org.partiql.lang.eval.bytesValue
+import org.partiql.lang.eval.dateValue
+import org.partiql.lang.eval.intValue
+import org.partiql.lang.eval.name
+import org.partiql.lang.eval.numberValue
+import org.partiql.lang.eval.stringValue
+import org.partiql.lang.eval.timeValue
+import org.partiql.lang.eval.timestampValue
+
+/**
+ * Only used to verify the actual `ExprValue` is the same as the expected `ExprValue` in test
+ *
+ * Ensures 2 [ExprValue]s to have the same type & value
+ */
+fun ExprValue.strictEquals(other: ExprValue): Boolean =
+    ExprValueStrictComparator.compare(this, other) == 0
+
+/**
+ * We implement this comparator since to compare 2 bags, we sort them first and then compare each item.
+ *
+ * The order defined in this comparator of [ExprValue] is not related to the order defined by [ExprValue.exprEquals()]
+ */
```

Comment:
/** 
A comparator defining an (arbitrary) total order on [ExprValue].

This comparator is a private implementation detail of [strictEquals]. 
It does not have any interesting semantic meaning, except for being able to compare any two values in a manner consistent with their strictEquals identity.  This is essential for ignoring element order when comparing bags and structs.
*/